### PR TITLE
cleanup development.md

### PR DESCRIPTION
**Pull request by lionelvillard**
## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -53,14 +49,13 @@ _Adding the `upstream` remote sets you up nicely for regularly
...
 
-Once you've [setup your development environment](#getting-started), install any
-of the sources _Github Source_, _AWS SQS Source_, _Camel Source_, _Kafka Source_
-with:
+Once you've [setup your development environment](#getting-started), install the
+_Github Source_ with:
 
 ```
-ko apply -f <source_name>/config  # e.g. github/config
+ko apply -f config
```
Comment (mattmoor) [inline DEVELOPMENT.md:58]: Format markdown:
```suggestion
ko apply -f config
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -85,11 +80,7 @@ kubectl -n knative-sources logs \
         -o name \
     )
 ```
-
-_See [camel/source/samples/README.md](./camel/source/samples/README.md),
-[kafka/source/README.md](./kafka/source/README.md) for instructions on
-installing the Camel Source and Kafka Source._
-
+
```
Comment (mattmoor) [inline DEVELOPMENT.md:83]: Format markdown:
```suggestion

```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**